### PR TITLE
unix, windows: add uv_fs_poll_getpath

### PR DESCRIPTION
**Pull request by saghul**
Equivalent of https://github.com/joyent/libuv/pull/1142 for fspoll handles.

/cc @indutny.

### CODE REVIEW

File: test/test-fs-poll.c [hunk 1]
```diff
@@ -144,3 +157,31 @@ static void poll_cb(uv_fs_poll_t* handle,
...
   return 0;
 }
+
+
+TEST_IMPL(fs_poll_getpath) {
+  char buf[1024];
+  size_t len;
+  loop = uv_default_loop();
+
+
```
Comment (indutny) [inline test/test-fs-poll.c]: Superfluos newline.

**Comment by indutny** [comment]
LGTM

**Comment by saghul** [comment]
Landed with the whitespace fixes. Thanks for the help reviewing Fedor!

**Comment by trevnorris** [comment]
I'm just curious about an architecture decision. Why not pass a uv_buf_t
instead of the char*?

**Comment by saghul** [comment]
Size is passed as a pointer, a buf takes the value.

**Pull request merged**